examples/debug/plugins/particle/main.pand

```diff
@@ -0,0 +1,24 @@
+use bamboo::{
+  particle,
+  util,
+}
+
+on init() {
+  bamboo::info("owo")
+}
+
+on interact(event, flow) {
+  let player = event.player 
+  let world  = @bamboo.default_world()
+
+  let particle = particle::Particle::new(
+    "witch",
+    player.pos(),
+    true,
+    util::FPos::new(0.0, 0.0, 0.0),
+    100,
+    0.0,
+  )
```

Comment:
I dislike having a function take 6 arguments. I think using a builder like this would be clearer:
```js
particle::Particle::builder("witch", player.pos())
  .long_distance(true)
  .velocity(0.0, 0.0, 0.0)
  .amount(100)
  .data(0.0)
  .build()
```
plus, the builder can have some defaults, so you don't need to set everything:
```js
particle::Particle::builder("witch", player.pos())
  .long_distance(true)
  .amount(100)
  .build()
```